trace.js

```diff
@@ -18,10 +18,12 @@ const traces = new Map();
 chain.filter.attach(function (error, frames) {
   return frames.filter(function (callSite) {
     const name = callSite && callSite.getFileName();
-    return (!name || name !== 'async_hooks.js');
```

Comment:
Yep, `internal/async_hooks.js` is missing. Please implement it as:

`return (!name || (name !== 'async_hooks.js` && name !== 'internal/async_hooks.js')`

I don't get why you would want to ignore `timers.js`. That can provide valuable information. If you don't want that look into my `clarify` module.

Reply:
Fixed